nens_auth_client/views.py

```diff
@@ -46,10 +46,10 @@ def login(request):
 
     The full flow goes as follows:
```

Comment:
Is wat hieronder staat helemaal in sync met de README?
Misschien hier anders naar de readme verwijzen om te zorgen dat er één bron van waarheid is?